packages/web/src/scss/components/TextArea/index.html

```diff
@@ -5,6 +5,10 @@
     <label for="textareaSimple" class="TextArea__label TextArea__label--required">Label</label>
     <textarea id="textareaSimple" class="TextArea__input" placeholder="Placeholder"></textarea>
   </div>
+  <div class="TextArea">
```

Comment:
I am missing `data-toggle="textArea"` for initializing the plugin.

Reply:
Oh, right. Fixed.